config/prometheus/monitor.yaml

```diff
@@ -12,7 +12,7 @@ metadata:
     app.kubernetes.io/part-of: jobset
     app.kubernetes.io/managed-by: kustomize
   name: controller-manager-metrics-monitor
-  namespace: system
```

Comment:
shouldn't this be overridden by kustomize?

Reply:
yes, after the reorganization of the config directory i just pushed, the override occurs in `config/prometheus/kustomization.yaml`. Users can apply prometheus manifests individually with `make prometheus`